frontend/libs/studio-components/src/components/StudioRecommendedNextAction/StudioRecommendedNextAction.tsx

```diff
@@ -0,0 +1,46 @@
+import { StudioButton } from '../StudioButton';
+import React from 'react';
+import classes from './StudioRecommendedNextAction.module.css';
+import { StudioCard } from '../StudioCard';
+
+export interface StudioRecommendedNextActionProps {
+  onSave: React.MouseEventHandler<HTMLButtonElement>;
+  saveButtonText: string;
+  onSkip: React.MouseEventHandler<HTMLButtonElement>;
+  skipButtonText: string;
+  title: string;
+  description: string;
+  hideSaveButton?: boolean;
+  children?: React.ReactNode;
+}
+
+export const StudioRecommendedNextAction = ({
+  onSave,
+  saveButtonText,
+  onSkip,
+  skipButtonText,
+  title,
+  description,
+  hideSaveButton = false,
+  children,
+}: StudioRecommendedNextActionProps): React.ReactElement => {
+  return (
+    <StudioCard>
+      <StudioCard.Header>{title}</StudioCard.Header>
+      <StudioCard.Content>
+        <p className={classes.description}>{description}</p>
+        {children}
+        <div className={classes.buttonGroup}>
+          {!hideSaveButton && (
```

Comment:
Should we always display the save button and provide a validation error if the user clicks it without entering a value in the input field? 🤔This might be a question for @Annikenkbrathen as well. 😊